tracee-rules/engine/engine.go

```diff
@@ -219,7 +234,7 @@ func (engine *Engine) LoadSignature(signature types.Signature) (string, error) {
 		engine.logger.Printf("error initializing signature %s: %v", metadata.Name, err)
 
 	}
-	go signatureStart(signature, c)
+	go signatureStart(signature, c, &engine.waitGroup)
```

Comment:
Shouldn't you add `engine.waitGroup.Add(1)` before this line?

Reply:
preferably inside signature start

Reply:
I am :)
Thanks